linera-views/README.md

```diff
@@ -11,6 +11,134 @@ atomically persisted in storage).
 
 See the [CONTRIBUTING](../CONTRIBUTING.md) file for how to help out.
 
+## Design
+
+# Possible clients
+
+We designed a `KeyValueStoreClient` trait that implements the basic functionalities
```

Comment:
"implements" is a weird for a trait. Maybe "represents"?

Reply:
"We have designed" is more consistent with the rest of the text in present tense (I think)

Reply:
Ok.